### ISSUE TITLE
Regenerate the parser

### ISSUE THREAD

**Issue by resolritter** [maxxnino/tree-sitter-zig#34]
The CI for `main` [is failing](https://github.com/maxxnino/tree-sitter-zig/actions/runs/4786391771/jobs/8510277973) because it's detecting that the parser files are outdated after #33.

I'll submit a follow-up PR for regenerating the parser.
